inst/templates/navbar.html

```diff
@@ -1,5 +1,5 @@
 {{#navbar}}
-<div class="navbar navbar-{{{type}}} navbar-fixed-top" role="navigation">
+<div class="navbar navbar-{{{type}}} navbar-fixed-top" role="navigation" id="topnavbar">
```

Comment:
Why do you need to add a new `#topnavbar` here? I think you can just change `#topnavbar` to `.navbar` in the JS code above.